kafka-daq-report/src/main.rs

```diff
@@ -77,6 +77,23 @@ type SharedData = Arc<Mutex<HashMap<u8, DigitiserData>>>;
 #[derive(Debug, Parser)]
 #[clap(author, version, about)]
 struct Cli {
+    #[clap(subcommand)]
+    command: Commands,
+}
+
+#[derive(Debug, Subcommand)]
+enum Commands {
+    #[clap(
+        name = "daq-trace",
+        about = "Provides metrics regarding data transmission from the digitisers via Kafka."
+    )]
+    DaqTrace(DaqTraceOpts),
+    #[clap(name = "message-debug", about = "Run message dumping tool.")]
+    MessageDebug(SharedOpts),
+}
+
+#[derive(Debug, Args)]
+struct SharedOpts {
```

Comment:
Could this use the term "common" in it's name rather than "shared". Sharing invokes a different concept in software and common is more accurate to what is actually meant here.